ci-operator/config/securesign/secure-sign-operator/README.md

```diff
@@ -0,0 +1,51 @@
+# securesign-secure-sign-operator-main-tests<!-- omit from toc -->
+
+## Table of Contents<!-- omit from toc -->
+- [General Information](#general-information)
+- [Purpose](#purpose)
+- [Process](#process)
+  - [Cluster Provisioning and Deprovisioning:](#cluster-provisioning-and-deprovisioning)
+  - [Test Setup, Execution, and Reporting Results](#test-setup-execution-and-reporting-results)
+- [Prerequisite(s)](#prerequisites)
+
+## General Information
+
+- **Repository**: [securesign-operator](https://github.com/securesign/secure-sign-operator)
+
+## Purpose
+
+To provision the necessary infrastructure and using that infrastructure to execute Trusted Artifact Signer (TAS) Operator E2E tests.
+The results of these tests will be reported to the appropriate sources following execution.
+
+## Process
+
+The TAS Operator scenario can be broken into the following basic steps:
+
+Testing with unreleased TAS operator with nightly OpenShift version and configurations:
+
+1. Build the TAS Operator image.
+2. Build the TAS Operator bundle.
+3. Provision an OpenShift cluster on AWS.
+4. Install the TAS Operator bundle built in the previous step.
+5. Run the TAS Operator tests.
+6. Gather the results.
+7. Deprovision the cluster. 
+
+### Cluster Provisioning and Deprovisioning:
+
+Please see the [`firewatch-ipi-aws`](https://steps.ci.openshift.org/workflow/firewatch-ipi-aws) documentation for more information on this workflow. This workflow is not maintained by the Interop QE team.
+
+### Test Setup, Execution, and Reporting Results
+
+## Prerequisite(s)
+
+### Environment Variables
+
+### Custom Images
+
+- `securesign-operator`
+  - [Containerfile](https://github.com/securesign/secure-sign-operator/blob/main/Dockerfile)
+  - This Dockerfile is used to build a container image for the TAS Operator. It starts with a Golang base image and sets up the workspace. It then copies the Go Modules manifests and source code files into the container. The build process includes installing dependencies, setting build arguments, and compiling the Go code into a binary called "securesign-operator." The final image is based on Red Hat Universal Base Image (UBI) and includes additional packages and setup steps. The entry point of the container is set to execute the "jaeger-operator" binary.
+- `securesign-bundle`
+  - [Containerfile](https://github.com/securesign/secure-sign-operator/blob/main/bundle.Dockerfile)
+  - This Dockerfile is used to build the Jaeger Operator bundle. It creates an image from scratch, indicating that it starts with an empty base. The labels define metadata for the bundle, including its media type, manifests, metadata, package, channels, and metrics. The COPY commands copy the bundle's manifests and metadata files to the appropriate locations as specified by the labels.
```

Comment:
just patch this up later. Bad copy paste

Reply:
done!